src/main/webapp/i18n/en/assessment.json

```diff
@@ -10,15 +10,22 @@
             "feedbackCommentPlaceholder": "You can enter feedback here...",
             "additionalFeedbackCommentPlaceholder": "You can enter additional feedback here...",
             "suggestion": {
+                "default": "Suggestion",
                 "suggested": "Suggestion (not accepted)",
```

Comment:
Maybe you can also just keep `Suggestion` here without anything else

Reply:
I think I'll adapt your idea for the German version and use `Open suggestion` here.